Cabal/Distribution/Simple/Utils.hs

```diff
@@ -777,7 +795,7 @@ createProcessWithEnv ::
 createProcessWithEnv verbosity path args mcwd menv inp out err = withFrozenCallStack $ do
     printRawCommandAndArgsAndEnv verbosity path args mcwd menv
     hFlush stdout
-    (inp', out', err', ph) <- createProcess $
+    (inp', out', err', ph) <- createProcess $ enableProcessJobs
```

Comment:
import `System.Process` qualified, and redefine local `createProcess` which would always `enableProcessJobs` if it seems to be the goal.